api/fees/data.go

```diff
@@ -16,10 +16,13 @@ import (
 	"github.com/vechain/thor/v2/tx"
 )
 
-const (
-	priorityNumberOfTxsPerBlock = 3
-	priorityPercentile          = 60
-)
+type Config struct {
```

Comment:
Do you think all these values can be turned into regular `int`? I see they are always cast to `int`

Reply:
Done, they are always positive but it is true that we save some casts and that it is also controlled by the flag itself.